bgpd/bgp_mplsvpn.c

```diff
@@ -1380,10 +1439,10 @@ static void vpn_policy_routemap_update(struct bgp *bgp, const char *rmap_name)
 				bgp->vpn_policy[afi]
 					.rmap[BGP_VPN_POLICY_DIR_FROMVPN] =
 					NULL;
-			}
-
-			vpn_leak_postchange(BGP_VPN_POLICY_DIR_FROMVPN, afi,
-					    bgp_get_default(), bgp);
+			} else
```

Comment:
I don't think this change is correct. `vpn_leak_postchange()` should be called always here.

Reply:
I am updating the PR to include #2083's way of doing this.